lessons/Early-Math/absolute-value-and-opposites-2/langs/en-us/index.adoc

```diff
@@ -0,0 +1,135 @@
+= Absolute Value and Opposites (Part 2)
+
+@description{Students analyze expressions and Circles of Evaluation that use both opposite and absolute value.}
+
+@lesson-prereqs{absolute-value-and-opposites}
+
+@keywords{Opposite, Absolute Value}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Students translate expressions with absolute value and opposite, using Circles, words and symbols.
+* Students analyze equivalence in pairs of Circles with absolute value and equivalence.
+* Students compute the result of expressions with absolute value and opposite.
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to about opposites and absolute value in combination.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* This activity involves a card sort. The easiest way to prepare for this is to give each pair of students an envelope containing the three sets of cards. Keep each set together with rubber bands or paperclips.
+
+|===
+
+
+== Translating with Absolute Value & Opposite @duration{25 minutes}
+
+=== Overview
+Students match Circles of Evaluation to expressions in words with absolute value and opposite, and then they consider how mathematical expressions correspond to those two representations.
+
+=== Launch
+
+Explain to students that they are going to receive @printable-exercise{three-sets-cards.adoc, three sets of cards}.
+
+The first set of cards includes Circles of Evaluation that use the `negate` and `num-abs` functions.
+
+The second set of cards translates those expressions into words.
+
+[.lesson-instruction]
+- With your partner, match each Circle card with the corresponding Words card.
```

Comment:
Curious whether Words card is serving us. 
I would describe the words as "Verbal Expressions". At the very least I think it would be valuable to at least sometime refer to them as expressions.

Reply:
I LOVE the idea of "verbal expressions". 100% agree with @flannery-denny that keeping the word "expression" is key